### DISCUSSION TITLE
"Options" column is coloured in data view but not in a table in a design view. How do I fix that?

### DISCUSSION THREAD

**Discussion by kpturner** [Budibase/budibase#11076] [Help] [answered]
In my imported schema I have a column that is of type "options" and so the default grid (in 'Data` as opposed to `Design`) colour codes the column like so
![image](https://github.com/Budibase/budibase/assets/308646/68f4b7a7-dcd0-46b5-a584-802e47ab9227)
When I design a table for this in the application, the table looks like this:
<img width="1305" alt="image" src="https://github.com/Budibase/budibase/assets/308646/2c6062d2-b3cb-4518-ae16-223c1e0f4b6d">
How does one configure the design table to behave like the default data table?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @kpturner 

Just to clarify, what you are referring to in the data view as a table isn't, its a grid component currently exclusive to the data area. Where as the one you see in design area is a table component. There is good news, within the next iteration of Budibase which will be released next week will include this new component being added to the design view. This means that you then will have a similar if not the same layout and functionality that you see within the data view.

I'm not sure if the table will receive an enhancement to it that would allow the options from the option's picker to be displayed in this way.

I hope this explanation is helpful and makes sense.
